client/src/App.tsx

```diff
@@ -1,12 +1,25 @@
 import React from "react";
 import "./App.css";
 import { AuthContainer } from "./containers";
+import CssBaseline from "@mui/material/CssBaseline";
+import { createTheme, ThemeProvider } from "@mui/material/styles";
+
+const theme = createTheme({
+  palette: {
+    secondary: {
+      main: "#f4003d",
```

Comment:
Dark mode as a stretch goal?

Reply:
Will add this to the projects board as a goal, but I think for now we should probably focus on just getting the UI correctly set up first and functioning.